src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTagger.cs

```diff
@@ -19,56 +23,95 @@ internal sealed class InlineParameterNameHintsTagger : ITagger<IntraTextAdornmen
     {
         private readonly ITagAggregator<InlineParameterNameHintDataTag> _tagAggregator;
         private readonly ITextBuffer _buffer;
+        private readonly ITextView _textView;
+        private readonly List<ITagSpan<IntraTextAdornmentTag>> _cache;
+        private ITextSnapshot _cacheSnapshot;
+        private readonly IClassificationFormatMap _formatMap;
+        private TextFormattingRunProperties _format;
+        private readonly IClassificationType _keyword;
 
         public event EventHandler<SnapshotSpanEventArgs> TagsChanged;
 
-        public InlineParameterNameHintsTagger(ITextBuffer buffer, ITagAggregator<InlineParameterNameHintDataTag> tagAggregator)
+        public InlineParameterNameHintsTagger(InlineParameterNameHintsTaggerProvider taggerProvider, ITextView textView, ITextBuffer buffer, ITagAggregator<InlineParameterNameHintDataTag> tagAggregator)
         {
+            _cache = new List<ITagSpan<IntraTextAdornmentTag>>();
```

Comment:
At this point since this file is now "complicated" let's #nullable enable the file. It'll make it clearer if those fields that are mutable can or can't be null.

Reply:
Done